src/posts/post.entity.ts

```diff
@@ -0,0 +1,39 @@
+import { BeforeInsert, Column, Entity, PrimaryGeneratedColumn } from "typeorm";
+import { Comment as CommentI } from "./interfaces/comment.interface";
+
+@Entity()
+export class Post {
+  @PrimaryGeneratedColumn("uuid")
+  id: string;
+
+  @Column()
+  title: String;
+
+  @Column()
+  content: String;
+
+  @Column()
+  image: String;
+
+  @Column()
+  dateOfCreation: String;
+
+  @Column({ default: "" })
+  dateOfModification: String;
+
+  @Column()
+  likes: Number;
+
+  @Column()
+  views: Number;
+
+  @Column({ type: "json" })
+  comments: Array<CommentI>;
```

Comment:
It's fine to do comments as JSON but how about we step up our game and do it OneToMany Table 🤔

Reply:
I just added it. You can check it out in the new PR.